File: UTC Time Zone/UTC_time_zone.py
#
#   Checks if time zone is Coordinated Universal Time (UTC), Python
#   Module written by Brandon Arvanaghi
#   Website: arvanaghi.com 
#   Twitter: @arvanaghi
#

import time

if time.tzname[0] == "Coordinated Universal Time" or time.tzname[1] == "Coordinated Universal Time":
	print("The time zone is Coordinated Universal Time (UTC), do not proceed.")
else:
	print("The time zone zone is " + str(time.tzname[time.daylight]) +". Proceed!")

File: UTC Time Zone/UTC_time_zone.cs
/*
    Checks if time zone is Coordinated Universal Time (UTC), C#
    Module written by Brandon Arvanaghi 
    Website: arvanaghi.com 
    Twitter: @arvanaghi
*/

using System;

namespace CheckTimezone {
    class Program {
        static void Main(string[] args) {
            if (TimeZone.CurrentTimeZone.StandardName == "Coordinated Universal Time") {
                Console.WriteLine("The time zone is Coordinated Universal Time (UTC), do not proceed.");
            } else {
                Console.WriteLine("The time zone is " + TimeZone.CurrentTimeZone.StandardName + ". Proceed!");
            }
            Console.ReadKey();
        }
    }
}